docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/Text/ProduceWordBags.cs

```diff
@@ -0,0 +1,89 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.ML.Data;
+using Microsoft.ML.Transforms.Text;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public static class ProduceWordBags
+    {
+        public static void Example()
+        {
+            // Create a new ML context, for ML.NET operations. It can be used for exception tracking and logging, 
+            // as well as the source of randomness.
+            var mlContext = new MLContext();
+
+            // Create a small dataset as an IEnumerable.
+            var samples = new List<TextData>()
+            {
+                new TextData(){ Text = "This is an example to compute bag-of-word features." },
+                new TextData(){ Text = "ML.NET's ProduceWordBags API produces bag-of-word features from input text." },
+                new TextData(){ Text = "It does so by first tokenizing text/string into words/tokens then " },
+                new TextData(){ Text = "computing Ngram and their neumeric values." },
+                new TextData(){ Text = "Each position in the output vector corresponds to a particular Ngram." },
```

Comment:
>Ngram [](start = 103, length = 5)

[https://en.wikipedia.org/wiki/N-gram](https://en.wikipedia.org/wiki/N-gram)
We call it NGram in code because you can't use `-` in variable, but for explanation maybe we should use `n-gram`?